### PR TITLE
better travis ci settings

### PR DESCRIPTION
**Pull request by supergibbs**

### CODE REVIEW

**Comment by supergibbs** [comment]
I closed #114 and squashed my commits into this new one. I tested a docker image build and it worked, need to test the full deploy still.

File: docker/upgrade_docker [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/expect
+
+spawn sudo apt-get -y install docker-engine
+
+expect "*** docker (Y/I/N/O/D/Z) \[default=N\] ?" 
+send "Y\r"
```
Comment (christinebrass) [inline docker/upgrade_docker:6]: :+1:

**Comment by cybrass** [comment]
:+1:

**Comment by christinebrass** [comment]
:+1:

**Comment by Krijger** [comment]
Hello @supergibbs . I saw your problem on http://blog.awolski.com/upgrade-docker-on-travis-ci/ , and I found a different option to fix it that I now use.

```
sudo apt-get -o Dpkg::Options::="--force-confnew" install -y docker-engine
```

You can see it working on https://travis-ci.org/stackwork/stackwork build '#144'

**Comment by supergibbs** [comment]
Thanks @Krijger! I actually got a response from someone at Travis CI and changed it to something similar: https://github.com/TrueCar/gluestick/blob/develop/.travis.yml#L16

**Pull request merged**